go/cmd/dolt/commands/log.go

```diff
@@ -117,6 +123,20 @@ func (cmd LogCmd) logWithLoggerFunc(ctx context.Context, commandStr string, args
 		return 1
 	}
 
+	opts, err := parseLogArgs(ctx, dEnv, apr)
+	if err != nil {
+		return HandleVErrAndExitCode(errhand.VerboseErrorFromError(err), usage)
+	}
+	if opts.commitSpec == nil {
+		opts.commitSpec = dEnv.RepoStateReader().CWBHeadSpec()
+	}
+	if len(opts.tableName) > 0 {
+		return handleErrAndExit(logTableCommits(ctx, dEnv, opts))
```

Comment:
Am I reading this wrong, or does this mean the tableName param is not accepted anymore?

Reply:
`tableName` is part `opts` now so using `opts.tableName` [here](https://github.com/dolthub/dolt/pull/4561/files#diff-c39bf5c66faf0dc0c5de474674ff85852e5d2043a181b9fcc4bffddeb69da453R438) instead of explicitly passing the tableName as an argument. Can change that back if you think it was clearer

Reply:
No worries; this makes sense. 👍